ROMFS/px4fmu_common/init.d/rc.rover_apps

```diff
@@ -5,6 +5,15 @@
 # NOTE: Script variables are declared/initialized/unset in the rcS script.
 #
 
+#
+# Start the rover interface CAN driver if enabled.
+#
+if param compare RI_ENABLE 1
```

Comment:
I'd drop this parameter. You can move this check under boards/ssrc/saluki-v1/init/... scripts, and just check the airframe (if param compare SYS_AUTOSTART 50005).

Also, could the type be detected then after establishing the connection (is the type even needed, or could it just be passed as a command line parameter here?)